scripts/checkpatch.pl

```diff
@@ -592,6 +592,20 @@ sub hash_show_words {
 	["__ATTR", 2],
 );
 
+our $api_defines = qr{(?x:
+	_ATFILE_SOURCE|
+	_BSD_SOURCE|
+	_DEFAULT_SOURCE
```

Comment:
@keith-packard missing pipe might be the culprit?

Reply:
It actually got moved to the end of the pattern when I sorted the symbols upon request from a reviewer... Checkout #60551 for the fix.